src/applications/personalization/profile360/components/AddressEditModal.jsx

```diff
@@ -1,24 +1,16 @@
 import React from 'react';
-import { kebabCase } from 'lodash';
```

Comment:
Do we need this behavior? We can pass it on down to the Vet360EditModal component as an ID and apply it there, but I didn't see what we were using it for.

Reply:
This was for analytics event names; okay with removing here. We can add back in as needed